src/content-helper/common/css/variables.scss

```diff
@@ -46,4 +47,5 @@ $html-font-size: 16px; // Used in functions.scss.
 	--font-size--extra-large: 1.2rem;
 	--black: #000;
 	--sidebar-black: #1e1e1e;
+	--parsely-green: #5ba745;
```

Comment:
Actually no, I haven't seen the original Parse.ly stylesheet. However, I got the color from the parse.ly SVG logo. Seems to be the same color, but because of consistency I will replace it with the hsl color.